python/alibiexplainer/alibiexplainer/explainer_method.py

```diff
@@ -0,0 +1,9 @@
+from typing import List, Dict
+
+class ExplainerMethodImpl(object):
```

Comment:
What's the point of defining this interface?

Reply:
So future different explainer implementations can be specified. At present there is just AnchorsTabular taken from Alibi.